src/main/antlr/Asciidoc.g4

```diff
@@ -48,7 +49,7 @@ word            : CHARS PUNCTUATION? ;
 CHARS           : [0-9a-zA-Z]+ ;
 IDENTIFIER      : [a-zA-Z]+ ;
 //BEGIN_PUNCTUATION : [¿¡] ; // Some languages have beginning punctuation (e.g. Spanish)
-PUNCTUATION     : [.:,;] ;
+PUNCTUATION     : '.' ;//[.:,;] ;
 //word            : ~[\n\t ]+ ;
-ANYCHAR         : ~[\n\t ]+ ; // PH
+ANY_CHAR         : ~[\n\t ] ; // PH
```

Comment:
Odd, with `ANY_CHAR: ~[\n\t ] ;` it does not parse the body for me
![1_original](https://cloud.githubusercontent.com/assets/5781153/22325482/8fc5ac2e-e3ae-11e6-8d18-9e36cb6a64f4.png)

But with it, it does
![2_with](https://cloud.githubusercontent.com/assets/5781153/22325484/9368cc4e-e3ae-11e6-969b-7407fdc66b0d.png)